graql/reasoner/atom/predicate/IdPredicate.java

```diff
@@ -130,16 +118,15 @@ public String toString() {
     /**
      * @return corresponding value predicate if transformation exists (id corresponds to an attribute concept)
      */
-    public ValuePredicate toValuePredicate() {
+    public ValuePredicate toValuePredicate(ConceptManager conceptManager) {
```

Comment:
Great! Looks like we're removing state that we don't need and passing it in, direction we want to go 💃